src/components/QuestionApprovalProgressBarSection.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from 'react';
+import { supabase } from '@/utils/supabaseClient';
+import { Container } from './Container';
+
+const QUESTION_COUNT_LIMIT = 100;
+
+export function QuestionApprovalProgressBarSection() {
+  const [approvedQuestionCount, setApprovedQuestionCount] = useState(0);
+
+  useEffect(() => {
+    const getApproveQuestionCount = async () => {
+      const { count, error } = await supabase
```

Comment:
This would only work if RLS is not enabled, right?

The tricky thing is, with the current implementation, it is enabled. It works just like this repo: https://github.com/ykdojo/supabase-nextjs-simple-auth-starter

For now, though, can we maybe just hard-code this? We have 3 questions approved at the moment.

Reply:
Ya, you're right. Let's hard-code this to 3 for now.